aetest/instance_vm.go

```diff
@@ -39,6 +39,10 @@ func NewInstance(opts *Options) (Instance, error) {
 			i.startupTimeout = opts.StartupTimeout
 		}
 	}
+	if len(os.Getenv("APPENGINE_DEV_APPSERVER_BINARY")) > 0 {
+		os.Setenv("APPENGINE_DEV_APPSERVER", os.Getenv("APPENGINE_DEV_APPSERVER_BINARY"))
+		PrepareDevAppserver = nil
```

Comment:
Are you sure this should be set to `nil`? This will disable whatever a user has set as a pre-launch hook.

Reply:
ah this is more of a problem with a file that is only internal to google.  i will find another way around `nilling` out the PrepareDevAppServer using our internal file.  Done.